pkgs/games/enyo-launcher/default.nix

```diff
@@ -1,25 +1,25 @@
 { mkDerivation, lib, fetchFromGitLab, cmake, qtbase }:
 
 mkDerivation rec {
-  pname = "enyo-doom";
-  version = "2.0.2";
+  pname = "enyo-launcher";
+  version = "2.0.5";
 
   src = fetchFromGitLab {
     owner = "sdcofer70";
-    repo = "enyo-doom";
+    repo = "enyo-launcher";
     rev = version;
-    sha256 = "1s1vpwrrpb9c7r2b0k1j7dlsfasfzmi6prcwql4mxwixrl7f8ms1";
+    sha256 = "sha256-qdVP5QN2t0GK4VBWuFGrnRfgamQDZGRjwaAe6TIK604=";
   };
 
   nativeBuildInputs = [ cmake ];
 
   buildInputs = [ qtbase ];
 
   meta = {
-    homepage = "https://gitlab.com/sdcofer70/enyo-doom";
+    homepage = "https://gitlab.com/sdcofer70/enyo-launcher";
     description = "Frontend for Doom engines";
     license = lib.licenses.gpl3Plus;
     platforms = lib.platforms.unix;
-    maintainers = [ lib.maintainers.tadfisher ];
+    maintainers = [ lib.maintainers.usrfriendly ];
```

Comment:
I approve of this, FWIW.

Reply:
Sorry, I didn't think to ask or anything as I'd assumed it was going to be a separate package.